feat: add docs about multiple Cypress image (#2255)
kubeshop/testkube: docs/executor-cypress.md

## **Creating the Testkube Test Script**

Create the Testkube test script from this example. The parameters passed are **repository**, **branch** and **the path where the project exists**. In the case of a mono repository, the parameters are **name** and **type**.

> We will use default Cypress executor (Testkube Cypress image).

Reviewer: ```suggestion
We will use the default Cypress executor (Testkube Cypress image).
```

Test execution completed in 1m45.405939s
```

> ## Using different Cypress images 

Reviewer: ```suggestion
## Using Different Cypress Images 
```

> In the Cypress world, usually you want to have control over your Runtime environment. Testkube can easily handle that for you! 

Reviewer: ```suggestion
In the Cypress world, there are instances when you want to have control over your Runtime environment. Testkube can easily handle that for you! 
```

We're building several Cypress images to handle features that different versions of Cypress can support.

> To use a different executor you can use one of our pre-built ones (for Cypress 8, 9, 10 and Custom Testkube imgae) or build your own Docker image based on a Cypress executor.

Reviewer: ```suggestion
To use a different executor you can use one of our pre-built ones (for Cypress 8, 9, 10 and Custom Testkube images) or build your own Docker image based on a Cypress executor.
```
